packages/core/src/shared/constants.ts

```diff
@@ -129,6 +129,7 @@ export const ecsIamPermissionsUrl = vscode.Uri.parse(
  */
 export const CLOUDWATCH_LOGS_SCHEME = 'aws-cwl' // eslint-disable-line @typescript-eslint/naming-convention
 export const AWS_SCHEME = 'aws' // eslint-disable-line @typescript-eslint/naming-convention
+export const EC2_LOGS_SCHEME = 'aws-ec2' // eslint-disable-line @typescript-eslint/naming-convention
```

Comment:
if this is a new symbol, it should use the naming convention. the old eslint-disable lines were just temporary to avoid churn.
```suggestion
export const ec2LogsScheme = 'aws-ec2'
```